src/CommandLine/Xml/DiagnosticXmlSerializer.cs

```diff
@@ -75,6 +75,8 @@ private static XElement SerializeDiagnostic(DiagnosticInfo diagnostic, IFormatPr
             new XAttribute("Id", diagnostic.Descriptor.Id),
             new XElement("Severity", diagnostic.Severity),
             new XElement("Message", diagnostic.Descriptor.Title.ToString(formatProvider)),
+            new XElement("Description", diagnostic.Descriptor.Description.ToString(formatProvider)),
+            new XElement("HelpLink", diagnostic.Descriptor.HelpLinkUri.ToString(formatProvider)),
```

Comment:
As these metadata are same for each diagnostic descriptor I would not put it here but I would put it in the summary (method `CreateSummary`).

Reply:
Done.